backend/capellacollab/settings/configuration/models.py

```diff
@@ -37,6 +38,62 @@ class MetadataConfiguration(core_pydantic.BaseModelStrict):
     environment: str = pydantic.Field(default="-", description="general")
 
 
+class BuiltInLinkItem(str, enum.Enum):
+    GRAFANA = "grafana"
+    PROMETHEUS = "prometheus"
+    DOCUMENTATION = "documentation"
+
+
+class UserRole(str, enum.Enum):
+    USER = "user"
+    ADMIN = "administrator"
+
+
+class BuiltInNavbarLink(core_pydantic.BaseModelStrict):
+    name: str
+    service: BuiltInLinkItem = pydantic.Field(
+        description="Built-in service to link to.",
+    )
+    role: UserRole = pydantic.Field(
+        description="Role required to see this link.",
+    )
+
+
+class CustomNavbarLink(core_pydantic.BaseModelStrict):
+    name: str
+    href: str = pydantic.Field(
+        description="URL to link to.",
+    )
+    role: UserRole = pydantic.Field(
+        description="Role required to see this link.",
+    )
+
+
+class NavbarConfiguration(core_pydantic.BaseModelStrict):
+    external_links: t.List[t.Union[BuiltInNavbarLink, CustomNavbarLink]] = (
```

Comment:
```suggestion
    external_links: list[BuiltInNavbarLink | CustomNavbarLink] = (
```

Use the new Python syntax as described in our code style: https://dsd-dbs.github.io/capella-collab-manager/development/backend/code-style/